README.md

```diff
@@ -1,70 +1,35 @@
-# Getting Started with Create React App
+# TodoList App Built with react
+A Todolist app built with basic functionalities like, add todo item, edit todo item delete item and mark item as completed.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+<!-- ## Demo version: []()  -->
 
-In the project directory, you can run:
+## Built With
+* React
 
-### `npm start`
+## Getting Started
+***
+Clone the repository to get a local copy
+From the cloned project directory
+Run `npm install` to install all dependencies
+Run `npm start` to start the local server
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+### Prerequisites
+* Node.JS
+* React
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+### Author
+***
+👤 Onyeagoziri Precious Akams
 
-### `npm test`
+* GitHub: [@kamzzy](https://github.com/kamzzy)
+* Twitter:[@precious_akams](https://twitter.com/precious_akams)
+* LinkedIn:[Onyeagoziri Akams](https://www.linkedin.com/in/onyeagoziri-akams/)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
 
-### `npm run build`
+Feel free to check the issues page.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
```

Comment:
It would much more readable if you got rid of the react default readme and created yours.